src/pages/loginAdmin/index.tsx

```diff
@@ -1,2 +1,91 @@
-const LoginAdmin = () => <div>LoginAdmin</div>;
+import { Container } from '@mui/material';
+import TextField from '@mui/material/TextField';
+import Box from '@mui/material/Box';
+import LoadingButton from '@mui/lab/LoadingButton';
+import { useFormik } from 'formik';
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useSnackbar, VariantType } from 'notistack';
+import { axiosInstance } from '../../utils/apis';
+import { formContainer, loginContainer } from './style';
+import adminSchema from './adminSchema';
+
+const LoginAdmin = () => {
+  const { enqueueSnackbar } = useSnackbar();
+  const [loading, setLoading] = useState(false);
+  const showSnackbar = (message:string, severity:VariantType) => {
+    enqueueSnackbar(message, { variant: severity });
+  };
+  const navigate = useNavigate();
+  const formik = useFormik({
+    initialValues: {
+      username: '',
+      password: '',
+    },
+    validationSchema: adminSchema,
+    onSubmit: async (values) => {
+      try {
+        setLoading(true);
+        await axiosInstance.post('/admin/login', {
+          data: {
+            username: values.username,
+        await axiosInstance.post('/admin/login', {
+          username: values.username,
+          password: values.password,
+        });
+        navigate('/admin');

```

Comment:
after loging in successfully, you will get access token from the API, please set access token to the local stoarge